bin/pytorch_inference/Main.cc

```diff
@@ -83,9 +83,10 @@ torch::Tensor infer(torch::jit::script::Module& module_,
 bool handleRequest(ml::torch::CCommandParser::CRequestCacheInterface& cache,
                    ml::torch::CCommandParser::SRequest request,
                    torch::jit::script::Module& module_,
+                   ml::core::CExecutor& executor,
                    ml::torch::CResultWriter& resultWriter) {
 
-    ml::core::async(ml::core::defaultAsyncExecutor(), [
+    ml::core::async(executor, [
```

Comment:
My inclination is to leave this using the default executor and not pass explicitly then update the start logic as before.